launch/sensor.launch

```diff
@@ -90,6 +93,16 @@
     doc="maximum number of attempts trying to communicate with the sensor.
          Counter resets upon successful connection"/>
 
+  <arg name="organized" default="true"
+    doc="generate an organzied point cloud"/>
+  <arg name="destagger" default="true"
+    doc="enable or disable point cloud destaggering"/>
+
+  <arg name="min_range" default="0.0"
+    doc="minimum lidar range to consider (meters)"/>
+  <arg name="max_range" default="1000.0"
```

Comment:
Why 1000? Can we set negative as unlimited or something? Seems weird to have an artificial limit

Reply:
1000 is well beyond the range of any of Ouster sensors current generations so this has no limit.
Negative values would require more handling and may or may not improve understanding, in any case I would be open to changing it if you and the rest feel strongly about it.

Reply:
Yeah, just ideally you would never have to update this value. If you set it to 10k maybe I'd agree, but 1000 is only like 3x our max range.

Also, if you use a key value (negative?) for disabled you can not even perform this comparison at all if the user doesn't care, saving performance.

Reply:
> Yeah, just ideally you would never have to update this value. If you set it to 10k maybe I'd agree, but 1000 is only like 3x our max range.

Surveying the net there does seem to be lidar sensor which can exceed 1k (although they are much bulkier than Ouster sensors), that being said I can increase the default max_range to 10k, not a big deal.

> Also, if you use a key value (negative?) for disabled you can not even perform this comparison at all if the user doesn't care, saving performance.

This would be implementation dependent, I mean currently I do the comparison in same pass to compute the cartesian so using negative values wouldn't help unless I bind a different operations. However, this is much more performant than the cli which needs two passes. I would rather keep as is as the gain would be minimal.